examples/user_input.tape

```diff
@@ -0,0 +1,21 @@
+# This is a vhs script. See https://github.com/charmbracelet/vhs for more info.
+# To run this script, install vhs and run `vhs ./examples/user_input.tape`
+Output "target/user_input.gif"
+Set Width 1200
+Set Height 600
+Hide
+Type "cargo run --example=user_input --features=crossterm"
+Enter
+Sleep 1s
+Show
+Sleep 2s
+Type e
+Sleep 1s
+Type "Hello, world!"
+Enter
+Sleep 2s
+Backspace 13
+Sleep 1s
+Type "Goodbye, world!"
+Enter
+Sleep 5s
```

Comment:
Should we add a lint check for checking if a new line exists at the end of a file?

Reply:
I tend to use `precommit` hooks locally and there's a `end-of-file-fixer`. I can look for something that we can run in CI if that's of interest.

Reply:
Yeah, go for it :)